charts/datadog/templates/containers-init-linux.yaml

```diff
@@ -37,7 +37,7 @@
       mountPath: {{ print "/host/" (dir (include "datadog.dockerOrCriSocketPath" .)) | clean }}
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
-    {{- if or .Values.datadog.systemProbe.enabled .Values.datadog.securityAgent.runtime.enabled }}
+    {{- if or .Values.datadog.systemProbe.enabled .Values.datadog.securityAgent.runtime.enabled $.Values.datadog.networkConfig.enabled }}
```

Comment:
I think `or` only works with 2 args.

Reply:
Also this test is used several time. It could be good to have a function in `_helpers.yaml` to remove the duplication.

Reply:
I did a simple test and render:

```
helloworld: {{ or false false true }}
```

and it seems that `or` accepts any number of args